client/modules/core/main.js

```diff
@@ -158,6 +159,46 @@ export default {
     return reactionState;
   },
 
+  /**
+   * absoluteUrl
+   * @summary a wrapper method for Meteor.absoluteUrl which sets the rootUrl to
+   * the current URL (instead of defaulting to ROOT_URL)
+   * @param {String} [path] A path to append to the root URL. Do not include a leading "`/`".
+   * @param {Object} [options]
+   * @param {Boolean} options.secure Create an HTTPS URL.
+   * @param {Boolean} options.replaceLocalhost Replace localhost with 127.0.0.1. Useful for services that don't recognize localhost as a domain name.
+   * @param {String} options.rootUrl Override the default ROOT_URL from the server environment. For example: "`http://foo.example.com`"
+   * @return {String} URL for the given path and options
+   */
+  absoluteUrl(path, options) {
```

Comment:
Generally, this is the right approach, wrapping the Meteor method in a Reaction method so that if we migrate away from Meteor, these APIs don't change.